plugins/module_utils/podman/podman_pod_lib.py

```diff
@@ -573,6 +574,8 @@ def running(self):
         """Return True if pod is running now."""
         if 'status' in self.info['State']:
             return self.info['State']['status'] == 'Running'
+        if 'status' in self.ps_info:
```

Comment:
let's run here `is 'status' in self.get_ps()`
Ans it will run only on old versions.
Also please add this reasoning in comment above it.

Reply:
I added a new commit to the pull request. Please let me know if it meets your requirements. I tried to find the best way to only fall through to the `get_ps()` if the other attempts to get the status failed.